### ISSUE TITLE
[12.0 -> 13.0] [base] Error while updating base module.

### ISSUE THREAD

**Issue by mojtabaakbarpour** [OCA/OpenUpgrade#2267]
Hi
When migrate base module from 12.0 to 13.0 get this error

```
2020-04-04 09:45:41,871 15700 ERROR odoo12 odoo.sql_db: bad query: UPDATE ir_model_data SET module = 'base', name = 'module_category_sales_point_of_sale' WHERE module = 'base' and name = 'module_category_point_of_sale'
ERROR: duplicate key value violates unique constraint "ir_model_data_module_name_uniq_index"
DETAIL:  Key (module, name)=(base, module_category_sales_point_of_sale) already exists.
 
2020-04-04 09:45:41,871 15700 ERROR odoo12 OpenUpgrade: Error after 0:00:00.003994 running UPDATE ir_model_data SET module = 'base', name = 'module_category_sales_point_of_sale' WHERE module = 'base' and name = 'module_category_point_of_sale' 
2020-04-04 09:45:41,871 15700 ERROR odoo12 OpenUpgrade: base: error in migration script base\migrations\13.0.1.3\pre-migration.py: duplicate key value violates unique constraint "ir_model_data_module_name_uniq_index"
DETAIL:  Key (module, name)=(base, module_category_sales_point_of_sale) already exists.
 
2020-04-04 09:45:41,872 15700 ERROR odoo12 OpenUpgrade: duplicate key value violates unique constraint "ir_model_data_module_name_uniq_index"
DETAIL:  Key (module, name)=(base, module_category_sales_point_of_sale) already exists.
```

**Comment by pedrobaeza** [comment]
Fix in #2269

**Comment by mojtabaakbarpour** [comment]
after this fix i got this error

`2020-04-11 04:16:45,813 15944 ERROR odoo12 OpenUpgrade: base: error in migration script base\migrations\13.0.1.3\pre-migration.py: 'ir.module.category' 
2020-04-11 04:16:45,814 15944 ERROR odoo12 OpenUpgrade: 'ir.module.category' 
Traceback (most recent call last):
  File "\openupgradelib\openupgrade.py", line 1778, in wrapped_function
    if use_env2 else cr, version)
  File "\OpenUpgrade\odoo\addons\base\migrations\13.0.1.3\pre-migration.py", line 750, in migrate
    rename_ir_module_category(env)
  File "\OpenUpgrade\odoo\addons\base\migrations\13.0.1.3\pre-migration.py", line 719, in rename_ir_module_category
    method="sql")
  File "\openupgradelib\openupgrade_merge_records.py", line 532, in merge_records
    record_ids = env[model_name].browse(record_ids).exists().ids
  File "\OpenUpgrade\odoo\api.py", line 463, in __getitem__
    return self.registry[model_name]._browse(self, (), ())
  File "\OpenUpgrade\odoo\modules\registry.py", line 178, in __getitem__
    return self.models[model_name]
KeyError: 'ir.module.category'
2020-04-11 04:16:45,835 15944 WARNING odoo12 odoo.modules.loading: Transient module states were reset 
2020-04-11 04:16:45,835 15944 ERROR odoo12 odoo.modules.registry: Failed to load registry 
Traceback (most recent call last):
  File "\OpenUpgrade\odoo\modules\registry.py", line 86, in new
    odoo.modules.load_modules(registry._db, force_demo, status, update_module)
  File "\OpenUpgrade\odoo\modules\loading.py", line 413, in load_modules
    report=report, models_to_check=models_to_check, upg_registry=upg_registry)
  File "\OpenUpgrade\odoo\modules\loading.py", line 190, in load_module_graph
    migrations.migrate_module(package, 'pre')
  File "\OpenUpgrade\odoo\modules\migration.py", line 185, in migrate_module
    migrate(self.cr, installed_version)
  File "\openupgradelib\openupgrade.py", line 1778, in wrapped_function
    if use_env2 else cr, version)
  File "\OpenUpgrade\odoo\addons\base\migrations\13.0.1.3\pre-migration.py", line 750, in migrate
    rename_ir_module_category(env)
  File "\OpenUpgrade\odoo\addons\base\migrations\13.0.1.3\pre-migration.py", line 719, in rename_ir_module_category
    method="sql")
  File "\openupgradelib\openupgrade_merge_records.py", line 532, in merge_records
    record_ids = env[model_name].browse(record_ids).exists().ids
  File "\OpenUpgrade\odoo\api.py", line 463, in __getitem__
    return self.registry[model_name]._browse(self, (), ())
  File "\OpenUpgrade\odoo\modules\registry.py", line 178, in __getitem__
    return self.models[model_name]
KeyError: 'ir.module.category'
2020-04-11 04:16:45,837 15944 CRITICAL odoo12 odoo.service.server: Failed to initialize database `odoo12`. 
Traceback (most recent call last):
  File "\OpenUpgrade\odoo\service\server.py", line 1179, in preload_registries
    registry = Registry.new(dbname, update_module=update_module)
  File "\OpenUpgrade\odoo\modules\registry.py", line 86, in new
    odoo.modules.load_modules(registry._db, force_demo, status, update_module)
  File "\OpenUpgrade\odoo\modules\loading.py", line 413, in load_modules
    report=report, models_to_check=models_to_check, upg_registry=upg_registry)
  File "\OpenUpgrade\odoo\modules\loading.py", line 190, in load_module_graph
    migrations.migrate_module(package, 'pre')
  File "\OpenUpgrade\odoo\modules\migration.py", line 185, in migrate_module
    migrate(self.cr, installed_version)
  File "\openupgradelib\openupgrade.py", line 1778, in wrapped_function
    if use_env2 else cr, version)
  File "\OpenUpgrade\odoo\addons\base\migrations\13.0.1.3\pre-migration.py", line 750, in migrate
    rename_ir_module_category(env)
  File "\OpenUpgrade\odoo\addons\base\migrations\13.0.1.3\pre-migration.py", line 719, in rename_ir_module_category
    method="sql")
  File "\openupgradelib\openupgrade_merge_records.py", line 532, in merge_records
    record_ids = env[model_name].browse(record_ids).exists().ids
  File "\OpenUpgrade\odoo\api.py", line 463, in __getitem__
    return self.registry[model_name]._browse(self, (), ())
  File "\OpenUpgrade\odoo\modules\registry.py", line 178, in __getitem__
    return self.models[model_name]
KeyError: 'ir.module.category'
`
